src/main/java/core/basesyntax/service/serviceimpl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.serviceimpl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportGenerator;
+
+public class ReportGeneratorImpl implements ReportGenerator {
```

Comment:
The first line after the class declaration should not be empty. Please remove the empty line.